compiler+runtime/src/cpp/jank/jit/processor.cpp

```diff
@@ -199,4 +238,54 @@ namespace jank::jit
     auto err(interpreter->ParseAndExecute({ s.data(), s.size() }));
     llvm::logAllUnhandledErrors(std::move(err), llvm::errs(), "error: ");
   }
+
+  option<native_persistent_string>
+  processor::find_dynamic_lib(native_persistent_string const &lib) const
+  {
+    auto const &lib_name{ shared_lib_name(lib) };
+    for(auto const &lib_dir : library_dirs)
+    {
+      auto lib_abs_path{ fmt::format("{}/{}", lib_dir.string(), lib_name) };
+      if(boost::filesystem::exists(lib_abs_path.c_str()))
+      {
+        return lib_abs_path;
+      }
+    }
+
+    return none;
+  }
+
+  result<void, native_persistent_string>
+  processor::load_dynamic_libs(native_vector<native_persistent_string> const &libs) const
+  {
+    for(auto const &lib : libs)
+    {
+      auto const &lib_abs_path{ boost::filesystem::absolute(lib.c_str()) };
```

Comment:
I think we want to be smarter about this. We can check `boost::filesystem:path{ lib }.is_absolute()` and then load it if it exists or fail. If it's a lib name, we don't want to just try to make it absolute, since that's going to try to load the lib name in the current directory, which may not actually be in the library dirs, right?